source/sass/blocks/how-it-works-item.scss

```
@@ -1,13 +1,66 @@
+@use "../variables.scss" as variables;
+
+
 .how-it-works-item {
   display: flex;
   align-items: center;
-}
-
-
-.how-it-works-item__text {
-  width: 200px;
-  margin-left: 20px;
+  gap: 20px;
+  position: relative;
   font-size: 14px;
   line-height: 129%;
   color: #444444;
+
+  &::before {
+    content: "";
+    width: 60px;
+    height: 60px;
+    background-repeat: no-repeat;
+    background-size: contain;
+    position: absolute;
```

```suggestion
    flex-shrink: 0;
```

Вместо `position: absolute;` предлагаю добавить запрет на сжатие картинки, сразу после этого сработает и `gap`